src/test/scala/uk/gov/hmrc/ui/specs/reportingnotifications/AddAndViewReportingNotificationJourneysSpec.scala

```diff
@@ -0,0 +1,192 @@
+      resultPage.heading shouldBe "Manage your digital platform reporting"
+    }
+
+    Scenario("View multiple operator notification with one RPO and one EPO Notification") {
+      Given("Organisation user is subscribed")
+      val enrolmentsData     = SubscriptionSteps.subscribedOrganisationEnrolment()
+      And("Organisation user logs in")
+      loginPage.show()
+      loginPage.loginAs(aUserCredentials.copy(enrolmentsData = enrolmentsData))
+      And("Already added the platform operator")
+      val platformOperatorId = PlatformOperatorSteps.addPlatformOperator()
+
+      When("Add multiple notifications")
+
+      indexPage.clickAddReportingNotification()
+      whichPlatformOperatorPage.continue()
+      PlatformNotificationStartPage(platformOperatorId).continue()
+      AddNotificationPage(platformOperatorId).selectReportingPlatformOperator().continue()
+      FirstPeriodPage(platformOperatorId).continue()
+      DueDiligencePage(platformOperatorId).selectExtendedTimeLimit().selectActiveSellerDue().continue()
+      NotificationCheckYourAnswersPage(platformOperatorId).continue()
+      NotificationSuccessPage(platformOperatorId).clickManageYourDigitalPlatformReporting()
+
+      indexPage.clickAddReportingNotification()
+      whichPlatformOperatorPage.continue()
+      NotificationViewPage(platformOperatorId).selectYes().continue()
+      PlatformNotificationStartPage(platformOperatorId).continue()
+      AddNotificationPage(platformOperatorId).selectExcludedPlatformOperator().continue()
+      FirstPeriodPage(platformOperatorId).continue()
+      NotificationCheckYourAnswersPage(platformOperatorId).continue()
+      NotificationSuccessPage(platformOperatorId).clickManageYourDigitalPlatformReporting()
+
+      Then("The result page should be 'Manage your digital platform reporting'")
+      resultPage.url       should include("/manage-reporting")
+      resultPage.heading shouldBe "Manage your digital platform reporting"
+
+      When("On view page select No")
+      indexPage.clickAddReportingNotification()
+      whichPlatformOperatorPage.continue()
+      NotificationViewPage(platformOperatorId).selectNo().continue()
+      ManageReportingPage(platformOperatorId).clickViewReportingNotification()
+      NotificationViewPage(platformOperatorId).selectNo().continue()
+
+      Then("The result page should be 'Manage your digital platform reporting'")
+      resultPage.url       should include("/manage-reporting")
+      resultPage.heading shouldBe "Manage your digital platform reporting"
+    }
+
+    Scenario("Add operator notification with EPO Notification where we have multiple operators") {
+      Given("Organisation user is subscribed")
+      val enrolmentsData      = SubscriptionSteps.subscribedOrganisationEnrolment()
+      And("Organisation user logs in")
+      loginPage.show()
+      loginPage.loginAs(aUserCredentials.copy(enrolmentsData = enrolmentsData))
+      And("Already added the platform operator")
```

Comment:
"Already added **more than one** platform operator"

Reply:
Here we only one platform operator. So text is fine.